src/Knp/Menu/MenuItem.php

```diff
@@ -458,7 +434,7 @@ public function removeChild($name): ItemInterface
 
         if (isset($this->children[$name])) {
             // unset the child and reset it so it looks independent
-            $this->children[$name]->setParent(null);
+            $this->children[$name]->setParent();
```

Comment:
I would keep passing `null` explicitly here. The intent of `setParent()` was to make the argument nullable, not optional (but on PHP 5.x, we add to make it optional to make it nullable with a typehint)

Reply:
Removing parameters with same value as default is an improvement suggested by phpstorm

Reply:
I'd rather keep the existing code here, which is more readable by making the intent clear.

Actually, I think a future major version of the library should make the argument required in the setter now that's possible (but it does not warrant doing a major version bump just for that change)